site/FAQ.php

```diff
@@ -1,41 +1,12 @@
 <html>
 	<?php
 		include_once('init.php');
-		$SiteTitle = 'St. Louis Theatre FAQ';
-		$PageTitle = 'FAQ';
-		PrintHeader($SiteTitle, $PageTitle);
+		PrintHeader('St. Louis Theatre FAQ', 'FAQ');
 	?>
-	<div class="Questions">
-		<p class="QuestionTitle">
-			What is the question?
-		</p>
-		<p class="QuestionAnswer">
-			This is an answer for the question listed above.
-		</p>
-		<p class="QuestionTitle">
-			What is the question?
-		</p>
-		<p class="QuestionAnswer">
-			This is an answer for the question listed above.
-		</p>
-		<p class="QuestionTitle">
-			What is the question?
-		</p>
-		<p class="QuestionAnswer">
-			This is an answer for the question listed above.
-		</p>
-		<p class="QuestionTitle">
-			What is the question?
-		</p>
-		<p class="QuestionAnswer">
-			This is an answer for the question listed above.
-		</p>
-		<p class="QuestionTitle">
-			What is the question?
-		</p>
-		<p class="QuestionAnswer">
-			This is an answer for the question listed above.
-		</p>
+	<div class='Questions')>
+		<?php
+			GetFaqQuestions();
```

Comment:
This is a step in the right direction, but even better would be to have this function return the array, but have the html printed in this page. The reason for this is that you might want to do different things with the FAQ questions array, and by printing html inside that function, you limit it to only helping for this one specific scenario.

Like in LACRM, think about how a `GetEvents` function might be used. It's used for the agenda on the Workspace, and agenda in the nav bar, the calendar view, and the agenda email that gets sent out each morning. All of those display the data in different ways, but they use the same underlying function for getting the events that need to be displayed.

So there's a single `GetEvents` function, and it returns raw data which is then echoed on this page.

Another angle to take when deciding what belongs in the function and what doesn't is to consider it like a front-end/back-end separation. The user actually loads the URL `FAQ.php` into their browser, and so this page is responsible for the front end (html, css, etc.). The files in the include folder are backend files which interact with the database, do complicated logic, etc. but they normally return data rather than echoing html directly.

Of course this somewhat contradicts what I already asked you to do with the header and footer function. Those are sort of exceptions because the exact same html will get printed from multiple different places. Using a function for that is ok, but you still want to separate the "front end" functions (e.g. `PrintFaqSection()`) from the "back end" functions (e.g. `GetFaqQuestions`). In that example, the front end function would call the back end function to get the raw data, but potentially other places might call that same front end function and use the data differently.